src/lib/components/event/event-history-timeline.svelte

```diff
@@ -349,15 +350,15 @@
   }
 
   :global(.vis-background, .vis-timeline) {
-    background-color: #18181b;
+    background-color: #000;
     color: white;
     border-radius: 0.75rem;
     border-width: 2px;
-    border: 2px solid #18181b;
+    border: 2px solid #000;
```

Comment:
Not needed for this PR, but we could also use Tailwind's `theme()` function. E.g. `theme('colors.black')`.

Reply:
There's also a lot of colors that could be updated here... I'm going to create a follow-up task to update this component a bit.